packages/react-dom-bindings/src/server/ReactDOMFloatServer.js

```diff
@@ -633,103 +633,105 @@ function titlePropsFromRawProps(
   return props;
 }
 
-export function resourcesFromElement(type: string, props: Props): boolean {
```

Comment:
I just broke this up into each sub-type. no need for the extra switch statement and it doesn't simplify any logic in any particular branch